ground/gcs/gcs.pro

```diff
@@ -20,4 +20,9 @@ unix:!macx:!isEmpty(copydata) {
 	QMAKE_EXTRA_TARGETS += bin_install
 }
 
+!equals(QT_MAJOR_VERSION, 5) | !equals(QT_MINOR_VERSION, 6) | !equals(QT_PATCH_VERSION, 1) {
+message("Cannot build dRonin GCS with Qt version $${QT_VERSION}.")

```

Comment:
I accidentally shipped artifacts before with the "wrong" patch level windows.  I prefer it be fatal for release builds at least.